_episodes/04-changes.md

```diff
@@ -474,6 +474,13 @@ Date:   Thu Aug 22 09:51:46 2013 -0400
 ~~~
 {: .output}
 
+> ## Word-based diffing
+>
+> Sometimes, e.g. in the case of the LaTeX documment a line-wise
+> diff is too coarse. That is where the `--color-words` option of
```

Comment:
I would rephrase it to explain a bit what color-words does. From the docs: "Highlights the changed words using colors". What do you think of changing your sentence "That is where ... " to include something similar?